### DISCUSSION TITLE
Specifying database when using MongoDB

### DISCUSSION THREAD

**Discussion by igorg-il** [jobrunr/jobrunr#378] [Q&A]
Hello
As I understand jobs are created in jobrunr database in jobs collection
We have few instances of the product which use the same mongo server. So they share the same jobrunr db. As a result one instance of our product can  create job and other execute it. We want to make sure the one who creates the job will execute it. 

So the solution may be to use different db for each but jobrunr always uses db named "jobrunr". 
Alternatively we can use different collection names per instance (under the same database) 

I saw there is parameter org.jobrunr.database.table_prefix but it looks like this parameter is ignored (all other parameters work as expected). This is also consistent to what I saw in code. MongoDBStorageProvider constructor contains lines:
            this.jobrunrDatabase = mongoClient.getDatabase("jobrunr");
            this.jobCollection = this.jobrunrDatabase.getCollection("jobs", Document.class);

Is there any way to config db/collection names or some opther suggested solution?

thank you

**Comment by igorg-il** [comment]
I saw now it was fixed in 4.0.0

**Reply by rdehuyss** [reply]
Indeed - it is. Please star the repo if you find JobRunr useful! Thx!
